state_machine.py

```diff
@@ -73,9 +75,60 @@ def update_user(email, password, reset_password_hash):
         return {'updated_user':updated_user}
     except Exception as e:
         raise Exception({'error':str(e.message)})
-    
-
 
+def get_user_by_user_id(user_id):
```

Comment:
move all document functions (like upsert document, get documents etc) to a new state_machine in new module (documents)